### ISSUE TITLE
enforce_content_type causes GET requests to return 415 status code

### ISSUE THREAD

**Issue by robmaceachern** [doorkeeper-gem/doorkeeper#1161]
### Steps to reproduce
- Enable `enforce_content_type` in `doorkeeper.rb`
- Attempt GET on `oauth/token/info` eg:
```
curl -X GET \
  http://localhost:3000/oauth/token/info \
  -H 'Authorization: Bearer 123'
```

### Expected behavior
The token information should be retrieved.

### Actual behavior
Responds with 415 status code.

Including `Content-Type: application/x-www-form-urlencoded` in the request allows the request to succeed.

Related to #1067. I don't think enforcing a content-type on GET requests is the correct behaviour since a GET request doesn't have a body. cc @baxang

**Comment by nbulaj** [comment]
Thanks @robmaceachern . Fixed in #1162 and will be released with the next version

**Comment by robmaceachern** [comment]
Thanks @nbulaj 👍
